js/style/style_layer.js

```diff
@@ -189,17 +190,15 @@ StyleLayer.prototype = util.inherit(Evented, {
 
         var name;
         for (name in declarations) { // apply new declarations
-            this.updatePaintTransition(name, declarations[name], options, globalOptions, animationLoop);
+            this.applyPaintDeclaration(name, declarations[name], options, globalOptions, animationLoop);
         }
         for (name in this._paintTransitions) {
             if (!(name in declarations)) // apply removed declarations
-                this.updatePaintTransition(name, null, options, globalOptions, animationLoop);
+                this.applyPaintDeclaration(name, null, options, globalOptions, animationLoop);
         }
-
-        this.updateConstantValues();
     },
 
-    updatePaintTransition: function (name, declaration, options, globalOptions, animationLoop) {
+    applyPaintDeclaration: function (name, declaration, options, globalOptions, animationLoop) {
```

Comment:
What's unclear to me is use of the word `apply`. Why `apply` here but `update` elsewhere?